Jenkinsfile

```diff
@@ -49,9 +49,12 @@ try {
             checkout scm
         }
 
+        stage('Check for updates') {
+            timeout(60) {
```

Comment:
well that's no way to write a Jenkinsfile 🤣

Reply:
doh.